frontend/src/components/ShootDetails/ShootCustomFieldsCard.vue

```diff
@@ -0,0 +1,69 @@
+<!--
+SPDX-FileCopyrightText: 2020 SAP SE or an SAP affiliate company and Gardener contributors
+
+SPDX-License-Identifier: Apache-2.0
+-->
+
+<template>
+  <v-card v-if="customFields">
+    <v-toolbar flat dark dense color="cyan darken-2">
+      <v-toolbar-title class="subtitle-1">Custom Fields</v-toolbar-title>
+    </v-toolbar>
+    <v-list>
+      <template v-for="(customField, index) in customFields" >
+        <v-divider v-if="index !== 0" inset :key="`${customField.key}-divider`"></v-divider>
+        <v-list-item :key="customField.key">
+          <v-list-item-icon>
+            <v-icon color="cyan darken-2" v-if="customField.icon">{{customField.icon}}</v-icon>
```

Comment:
Shouldn't the v-if be in the `<v-list-item-icon>` tag?

Reply:
The name of the first component was `CustomField`. Why is this component named  `ShootCustomFieldsCard` and not `CustomFieldsCard`

Reply:
> Shouldn't the v-if be in the `<v-list-item-icon>` tag?

then we would need to add a padding, otherwise it wouldn't be aligned anymore, that's why I haven't put the v-if on the `v-list-item-icon` tag

Reply:
> The name of the first component was `CustomField`. Why is this component named `ShootCustomFieldsCard` and not `CustomFieldsCard`

I think we should rather add a prefix to the CustomField component. Maybe we want to introduce custom fields for other resources, like projects etcs. That was also the main reason why I prefixed this component name